test/powershell/engine/Basic/Credential.Tests.ps1

```diff
@@ -0,0 +1,6 @@
+Describe "Credential tests" -Tags "CI" {
+    It "Explicit cast for an empty credential returns null" {
+         # We should explicitly check that the expression returns $null
+         [PSCredential]::Empty.GetNetworkCredential() -eq $null | Should Be $true
```

Comment:
Shouldn't the test be: "[PSCredential]::Empty.GetNetworkCredential() | Should Be $Null"?

Reply:
A agree this would make any error, have more information to debug the issue.  Therefore making a rerun less likely to be needed.

Reply:
Fixed.